sample/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,7 @@
 
     <string name="hello_world">Hello world!</string>
     <string name="action_settings">Settings</string>
+
+    <string name="mapboxAccessToken"></string>
```

Comment:
I've added an AirMapView Sample App specific Access Token and Mapbox Streets Map Id so that Mapbox Web maps will work out of the box in the Sample App.  I also added a note in `strings.xml` to tell people how they can get their own free Mapbox maps and credentials by signing up at https://www.mapbox.com/signup/ .